fidesctl/src/fidesctl/core/export.py

```diff
@@ -126,13 +126,13 @@ def generate_system_records(
     server_system_list: List,
     url: str,
     headers: Dict[str, str],
-) -> List[Tuple[str, str, str, str, str, str, str, str, str, str, str, str]]:
+) -> List[Tuple[str, ...]]:
```

Comment:
I'm not sure this is an improvement, but it makes it a bit cleaner. Open to reverting to being very explicit about the length of the tuple if that is preferred 👍🏽

Reply:
Thanks! That all makes sense to me 🙌🏽